dissect/target/tools/query.py

```diff
@@ -230,15 +280,23 @@ def main():
         if len(record_entries):
             rs = record_output(args.strings, args.json)
             for entry in record_entries:
-                for record_entries in entry:
-                    if args.hash:
-                        rs.write(hashutil.hash_uri_records(target, record_entries))
+                try:
+                    for record_entries in entry:
+                        if args.hash:
+                            rs.write(hashutil.hash_uri_records(target, record_entries))
+                        else:
+                            rs.write(record_entries)
+                        count += 1
+                        if args.limit is not None and count >= args.limit:
+                            break_out = True
+                            break
+                except Exception as problem:
+                    # Ignore errors if multiple functions
+                    if len(funcs) > 1:
+                        target.log.error(f"Exception occurred while processing output of {func}", exc_info=problem)
```

Comment:
```suggestion
                except Exception as exc:
                    # Ignore errors if multiple functions
                    if len(funcs) > 1:
                        target.log.error("Exception occurred while processing output of %s", func, exc_info=exc)
```
Or `e` is popularly used in the codebase.

Reply:
Also why was this try/except actually added? I don't think anything changed here?

Reply:
If you run multiple functions, through the use of a wild card I assumed you don't want to stop if one of those in the set causes a problem?

Reply:
Ahh nevermind, I read the code wrong. I didn't realize that `entry` would be the generator from the plugins themselves. Carry on.